Move prover returns: exiting with condition generation errors
error: [boogie translator] Tuple not yet supported
  ┌─ tests/sources/functional/restrictions.move:9:32
  │
9 │         fun f1(): (u64, u64) { (1u64, 2u64) }
  │                                ^^^^^^^^^^^^

error: [boogie translator] `|x|e` (lambda) currently only supported as argument for `all` or `any`
   ┌─ tests/sources/functional/restrictions.move:12:28
   │
12 │         fun f2(): | |num { | | 1 }
   │                            ^^^^^

error: [boogie translator] Invoke not yet supported
   ┌─ tests/sources/functional/restrictions.move:16:13
   │
16 │             f(1u64)
   │             ^^^^^^^
